_posts/2016-10-16-getting-started-in-scala-on-android.md

```diff
@@ -0,0 +1,217 @@
+---
+layout: post
+asset-type: post
+name: getting-started-in-scala-on-android
+title: Getting started in Scala on Android
```

Comment:
I'm even thinking about changing the title to "Setting up Scala on Android", as "Getting started" could make someone expect to see some code.